source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -124,17 +132,22 @@ SimpleHttpCache::Entry SimpleHttpCache::lookup(const LookupRequest& request) {
   if (VaryHeader::hasVary(*iter->second.response_headers_)) {
     return varyLookup(request, iter->second.response_headers_);
   } else {
+    Http::ResponseTrailerMapPtr trailers_map = nullptr;
```

Comment:
you don't need to init this; unique_ptr has a constructor which has the same effect.